build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+        maven("https://developer.huawei.com/repo/")
+        maven("https://maven.pkg.github.com/Rocket-Beer/*") {
+            println("**** Rocket Beer maven ****")
+
+            credentials {
+                val userName = publish.CommonMethods.getPublisherUserName(rootProject)
+                val userPass = publish.CommonMethods.getPublisherPassword(rootProject)
+
+                println("user = $userName :: password = $userPass")
```

Comment:
`println` just for tests, if it works don't pull this code, this are credentials

Reply:
True I will remove this.